src/panels/CreateFleetPanel.ts

```diff
@@ -1,4 +1,124 @@
 import { ContainerServiceFleetClient, Fleet } from "@azure/arm-containerservicefleet";
+import { BasePanel, PanelDataProvider } from "./BasePanel";
+import { Uri, window } from "vscode";
+import { MessageHandler, MessageSink } from "../webview-contract/messaging";
+import {
+    InitialState,
+    ProgressEventType,
+    ToVsCodeMsgDef,
+    ToWebViewMsgDef,
+} from "../webview-contract/webviewDefinitions/createFleet";
+import { TelemetryDefinition } from "../webview-contract/webviewTypes";
+import { ResourceManagementClient } from "@azure/arm-resources";
+import { ReadyAzureSessionProvider } from "../auth/types";
+import { getAksFleetClient, getResourceManagementClient } from "../commands/utils/arm";
+import { getResourceGroups } from "../commands/utils/resourceGroups";
+import { failed } from "../commands/utils/errorable";
+
+const contentId = "createFleet";
+
+export class CreateFleetPanel extends BasePanel<typeof contentId> {
```

Comment:
it should be extends BasePanel<"createFleet"> would work, base panel is abstract class that has the basic methods for a panel and data providers.

Reply:
this type refers to the one you have defined it in webviewTypes.ts file @JunyuQian 

![image](https://github.com/user-attachments/assets/79221412-3061-4b68-b09b-ced38ec986af)